src/main/java/org/apache/sling/discovery/base/connectors/announcement/package-info.java

```diff
@@ -23,8 +23,7 @@
  *
  * @version 1.1.2
  */
-@Version("2.0.0")
+@Version("3.0.0")
```

Comment:
Switching annotations (from Bnd to OSGi only requires a micro(!) version bump, i.e. 2.0.1 is totally fine.